test/language/number/scientific_missing_fractional_part.wren

```diff
@@ -0,0 +1 @@
+var x = 1.e // expect error

```

Comment:
I've just updated the file (I didn't understood that "expect error" was only for compile time errors), but the test still fails. And I have no clue why. The test does pass on my machine.